### PR TITLE
Initial Add of Getting Started Presentation

### PR DESCRIPTION
**Pull request by timodwhit**
@arknoll and @rmcveigh can you check out the presentation. 

To see it, you'll need to have node, npm installed. Run bower install, and then open presentations/getting-started/index.html in your browser

Feel free to comment on the HTML! It'll be a fun experiment

### CODE REVIEW

File: presentations/getting-started/index.html [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+          </p>
+          <p>
+            By <span class="fragment highlight-blue">integrating regularly</span>, you can  <span class="fragment highlight-blue">detect errors quickly</span>, and locate them <span class="fragment highlight-blue">more easily.</span>"
+          </p>
+          <p>
+            -- <a href="http://www.thoughtworks.com/continuous-integration">Thoughtworks.com </a>
+        </section>
+        <section>
+          <h2>What is Vampd?</h2>
+          <p>VAMPD is tool to ease continuous integration by easing and standardizing local development setup.</p>
```
Comment (bxtaylor) [inline presentations/getting-started/index.html]: Should be "VAMPD is a tool"

Comment (bxtaylor) [inline presentations/getting-started/index.html]: In fact, I would edit it down to "VAMPD is a tool to ease continuous integration by standardizing the local development setup." Both ease and easing is redundant.

File: presentations/getting-started/index.html [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+          <p><small>VAMPD is a fork of no longer existent drupal-lamp provisioner from Newmedia.</small></p>
```
Comment (bxtaylor) [inline presentations/getting-started/index.html]: Sounds awkward - would edit it to "VAMPD is a fork of drupal-lamp, the now defunct provisioner from Newmedia."

File: presentations/getting-started/index.html [hunk 3]
```diff
@@ -0,0 +1,391 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+
+    <title>VAMPD - Getting Started.</title>
+
+    <meta name="description" content="A toold for continuous integration on a local envrionment">
+    <meta name="author" content="Hakim El Hattab">
```
Comment (arknoll) [inline presentations/getting-started/index.html]: Who is this Hakim El Hattab and how did he write this presentation?

File: presentations/getting-started/index.html (hunk 2 above)
Comment (arknoll) [inline presentations/getting-started/index.html]: I would soften it even more. "VAMPD is a fork of drupal-lamp." Or even remove the line altogether.

File: presentations/getting-started/index.html [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+        <section>
+          <h2>What is Vampd?</h2>
+          <p>VAMPD is tool to ease continuous integration by easing and standardizing local development setup.</p>
+          <p><small>VAMPD is a fork of no longer existent drupal-lamp provisioner from Newmedia.</small></p>
+        </section>
+
+        <section>
+          <h2>Goals of VAMPD and CI</h2>
+          <ul>
+            <li>Common Development Practices</li>
```
Comment (arknoll) [inline presentations/getting-started/index.html]: Promote Best Practice Drupal Development

File: presentations/getting-started/index.html [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+            <li>Share Repos</li>
```
Comment (arknoll) [inline presentations/getting-started/index.html]: What does this mean?

File: presentations/getting-started/index.html [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+            <li>Integrate Regularly</li>
+            <li>Detect Errors Quickly</li>
+            <li>Ease Local Development</li>
+          </ul>
+        </section>
```
Comment (arknoll) [inline presentations/getting-started/index.html:78]: I would say:
1. Standardize local environments
1. Promote Best Practice Drupal Development

File: presentations/getting-started/index.html [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+            <li>Server admins don't work there</li>
+          </ul>
+        </section>
+
+        <section>
+          <h2>How do we address this?</h2>
+          <ul>
+            <li>Vagrant/Virtual Box</li>
+            <li>Reproducible Provisioner</li>
+            <li>Force Testing</li>
```
Comment (arknoll) [inline presentations/getting-started/index.html]: What does this mean? Force testing?

File: presentations/getting-started/index.html [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+          <p><em>Know what will happen</em></p>
+        </section>
+
+        <section>
+          <h2>How to do that in Drupal?</h2>
+          <ul>
+            <li>hook_update_N</li>
+            <li>Features</li>
+            <li>CINC</li>
+            <li>Install profiles</li>
```
Comment (arknoll) [inline presentations/getting-started/index.html:120]: Add Drush make?

**Comment by arknoll** [comment]
I would add a slide or two about what it actually looks like to develop while using a headless virtualbox. Many people may not realize that you aren't actually logging into the vm.

File: presentations/getting-started/index.html (hunk 1 above)
Comment (timodwhit) [inline presentations/getting-started/index.html]: Agreed. Thank you.

File: presentations/getting-started/index.html (hunk 2 above)
Comment (timodwhit) [inline presentations/getting-started/index.html]: Good thought. Thank you both

File: presentations/getting-started/index.html (hunk 3 above)
Comment (timodwhit) [inline presentations/getting-started/index.html]: editted.

File: presentations/getting-started/index.html (hunk 5 above)
Comment (timodwhit) [inline presentations/getting-started/index.html]: Changed to: "VAMPD is a tool to ease continuous integration by standardizing and automating the local development setup."

File: presentations/getting-started/index.html (hunk 7 above)
Comment (timodwhit) [inline presentations/getting-started/index.html]: How about: Allow testing through automation,

The idea being that you have to make sure your code will work on install or update before going live. If I pull your code, I have to be able to have run update.php or an install and run with it.

**Comment by timodwhit** [comment]
Thank you for the suggestions both, I have added a couple slides at the end, that talk about actually developing with it, cutting it down to about 3-5 steps, which is cool. 

So hopefully I can drive home how easy it is.

**Comment by timodwhit** [comment]
Gave the presentation at the boulder dbug. There is still more work to be done to it. But for now. This will be the initial start

**Pull request merged**